lib/tasks/update_qualified_teacher_status_for_profiles.rake

```diff
@@ -0,0 +1,10 @@
+namespace :jobseeker_profile do
+  desc "Update qualified_teacher_status from 'non_teacher' to 'no'"
+  task update_non_teacher_status: :environment do
+    profiles_to_update = JobseekerProfile.where(qualified_teacher_status: :non_teacher)
+
+    profiles_to_update.each do |profile|
+      profile.update(qualified_teacher_status: :no)
+    end
```

Comment:
Suggestion:

Unless there is a need to instantiate the profiles in memory one by one we could do this with a single SQL query that would be way faster.

```suggestion
    JobseekerProfile.where(qualified_teacher_status: :non_teacher)
                    .update_all(qualified_teacher_status: :no)
```